src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -442,11 +443,20 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      * @param {Error} e - API error
      * @return {void}
      */
-    fetchFeedItemsErrorCallback = (feedItems: FeedItems): void => {
+    fetchFeedItemsErrorCallback = (feedItems: FeedItems, errors: ElementsXhrError[]): void => {
+        const { onError } = this.props;
+
         this.setState({
             feedItems,
             activityFeedError: activityFeedInlineError,
         });
+
+        if (errors.length) {
+            onError(errors[0], ERROR_CODE_UNKNOWN, {
```

Comment:
It seems like this will be difficult to differentiate the parent application, since the error will be dynamic. Should we create a new error object and code, such as `ERROR_CODE_FETCH_ACTIVITY`, instead?